dynamiq/connections/connections.py

```diff
@@ -934,6 +934,111 @@ def connect(self):
         return super().connect()
 
 
+class Elasticsearch(BaseConnection):
+    """
+    Represents a connection to the Elasticsearch service.
+
+    Attributes:
+        url (str): The URL of the Elasticsearch service
+        api_key (str): API key for authentication
+        username (str): Username for basic authentication
+        password (str): Password for basic authentication
+        cloud_id (str): Cloud ID for Elastic Cloud deployment
+        ca_path (str): Path to CA certificate for SSL verification
+        verify_certs (bool): Whether to verify SSL certificates
+        use_ssl (bool): Whether to use SSL for connection
+    """
+
+    url: str = Field(default_factory=partial(get_env_var, "ELASTICSEARCH_URL", "http://localhost:9200"))
```

Comment:
Let's remove the default value, as connecting to the cloud is possible using the cloud_id without a URL. Keeping it as it is does not provide proper debugging, as it suggests we are connecting to localhost.
```suggestion
    url: str = Field(default_factory=partial(get_env_var, "ELASTICSEARCH_URL", None))
```